src/Admin/AbstractAdmin.php

```diff
@@ -3226,6 +3250,15 @@ protected function configureDefaultFilterValues(array &$filterValues)
     {
     }
 
+    /**
+     * Configures a list of form options.
+     *
+     * @param array<string, mixed> $formOptions
+     */
+    protected function configureFormOptions(array &$formOptions)
```

Comment:
Should we introduce a `FormOptions` class instead of an referenced array? This will help to use more specific extension points.

Reply:
I don't think so, all the options allowed are coming from Symfony.
If the interface should exist it should be in the Symfony/form project.

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/AbstractAdmin.php#L1261
https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/baf4b1281b78c5b4bdbc229d077a1cb0a5e8c045/src/Builder/FormContractor.php#L97